hooks/pre/25-remove_apache_from_foreman_group.rb

```diff
@@ -5,7 +5,7 @@ def remove_user_from_group(user, group)
     is_member = false
   end
   if is_member
-    execute("gpasswd -d '#{user}' '#{group}'")
+    execute!("gpasswd -d '#{user}' '#{group}'")
```

Comment:
I wonder if this is still needed. It was done in 1.23 to fix upgrades, but it's probably safe to drop by now.